python/testpluscodes.py

```diff
@@ -0,0 +1,53 @@
+import pluscodes
```

Comment:
I'd really like the tests to dynamically read in the test data from the test_data directory. This ensures that if we add new test cases, that they will get run on the different languages. Can you do that?